File: src/main/java/team4/footwithme/team/repository/TeamRateRepository.java
package team4.footwithme.team.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import team4.footwithme.team.domain.TeamRate;

@Repository
public interface TeamRateRepository extends JpaRepository<TeamRate, Long> {
}

File: src/main/java/team4/footwithme/team/repository/TeamRateRepository.kt
package team4.footwithme.team.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import team4.footwithme.team.domain.Team
import team4.footwithme.team.domain.TeamRate

@Repository
interface TeamRateRepository : JpaRepository<TeamRate?, Long?> {
    fun findEvaluationsByTeam(team: Team?): List<TeamRate?>?
}
